solidity/test/utils/TestLockUtils.js

```diff
@@ -0,0 +1,64 @@
+import {createSnapshot, restoreSnapshot} from '../helpers/snapshot';
+
+const BN = web3.utils.BN
+const chai = require('chai')
+chai.use(require('bn-chai')(BN))
+const expect = chai.expect
+
+const LockStub = artifacts.require('./stubs/LockStub.sol');
+
+contract('LockStub', (accounts) => {
```

Comment:
We should have `contract('LockUtils'` here. We are testing the library, the fact we need to do it through a stub contract is just an implementation detail.